<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagement</class>
 <widget class="QDialog" name="UserManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AvisionR - User Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>16</width>
     <height>521</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_newUser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>On the Clock:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Double click to Edit, Delete</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_done">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Users">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>501</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;

}

QTableCornerButton::section 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;
}

color:rgb(255, 255, 255);
background-color: #262626;
</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_forceCLockOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Force Clock Out</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_clock">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>271</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;

}

QTableCornerButton::section 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;
}

color:rgb(255, 255, 255);
background-color: #262626;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
